docs/source/core_services/service_agents/file_watch_publisher/FileWatchPublisher-Agent.rst

```diff
@@ -0,0 +1,30 @@
+FileWatchPulblisher Agent
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+
+Introduction
+============
+
+FileWatchPublisher agent watches files for changes and publishes those changes per line on the corresponding topics.
+Files and topics should be provided in the configuration.
+
+Configuration
+-------------
+
+A simple configuration for FileWatchPublisher with two files to monitor is as follows:
+
+::
+
+    [
```

Comment:
Should the config file be an object itself with the array as a property value?  I know it's valid json but it seems strange not to have the outer { in configurations.